stock_valuation_account_manual_adjustment/wizards/mass_create.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, exceptions, fields, models, _
+
+
+class StockValuationAccountMassAdjust(models.TransientModel):
+    _name = 'stock.valuation.account.mass.adjust'
+    _description = 'Mass Adjust Stock Valuation Account Discrepancies'
+
+    @api.model
+    def _default_journal(self):
+        res = self.env['account.journal'].search([('type', '=', 'general')])
+        return res and res[0] or False
+
+    increase_account_id = fields.Many2one(
+        comodel_name='account.account',
+        string='Valuation Increase Contra-Account')
+    decrease_account_id = fields.Many2one(
+        comodel_name='account.account',
+        string='Valuation Decrease Contra-Account')
```

Comment:
I think those fields should be required. In other case we get the following:

![image](https://cloud.githubusercontent.com/assets/19620251/24245945/3e2799ac-0fc5-11e7-9ef1-8fcadc5c8fbb.png)